.gitlab-ci.yml

```
@@ -859,45 +854,39 @@ workflow:
 .on_kitchen_tests_a6:
   - <<: *if_not_version_6
     when: never
-  - <<: *if_kitchen
+  - <<: *if_installer_tests
 
 .on_kitchen_tests_a6_always:
```

```suggestion
.on_installer_e2e_tests_a6_always:
```
argh, sorry, I cannot stop myself from doing this suggestion :D

---

I promise I will do a follow up PR to fix this :D 
https://datadoghq.atlassian.net/jira/software/c/projects/APL/boards/5339